config/locales/en.yml

```diff
@@ -29,12 +29,8 @@ en:
       contact: "Contact"
 
   home:
-    blurb:
-      intro: "%{site_name} is a community of food gardeners. We're building an open source platform to help you learn about growing food, track what you plant and harvest, and swap seeds and produce with other gardeners near you."
-      perks: "Join now for your free garden journal, seed sharing, forums, and more."
-      sign_up: "Sign up"
-      already_html: "Or %{sign_in} if you already have an account"
-      sign_in_linktext: "sign in"
+    home_title:
+      tagline: "A community of food growers"
```

Comment:
This extremely creative tagline isn't necessarily my suggestion for what the tagline should be. I just needed something there to test locale changing :)

Does anyone have any interesting ideas for this one?